helm/prometheus-meta-operator/templates/prometheus-rules/azure.management-cluster.rules.yml

```diff
@@ -15,16 +15,26 @@ spec:
       annotations:
         description: '{{`Cluster {{ $labels.cluster_id }} creation is taking longer than expected.`}}'
         opsrecipe: cluster-creation-failed/
-      # statusresource_cluster_status{app="azure-operator.*", status="Creating"} give us a vector with all cluster statuses
-      # in Azure clusters which stays at 'Creating' state.
-      expr: (statusresource_cluster_status{app=~"azure-operator.*", status="Creating"} == 1 or cluster_operator_cluster_status{app=~"cluster-operator.*", status="Updating", provider="azure"} == 1)
+      expr: azure_operator_cluster_status(status="Creating"} == 1
       for: 40m
       labels:
         area: kaas
         cancel_if_outside_working_hours: "true"
         severity: notify
         team: celestial
         topic: azure
+    - alert: AzureClusterUpdateFailed
```

Comment:
Would it make sense to unify the terminology and `s/update/upgrade/` ?

We usually talk about upgrades when the cluster version is bumped up. Updating cluster gives me an association to cluster configuration change, such as node count change, node pool creation / deletion, node size change etc. that doesn't affect the used release.

WDYT?

Reply:
renamed